internal/cmd/io_npipe.go

```diff
@@ -136,3 +138,42 @@ func (nio *npipeio) StderrPath() string {
 func (nio *npipeio) Terminal() bool {
 	return nio.terminal
 }
+
+// CreatePipeAndListen is a helper function to create a pipe listener
+// and accept connections. Returns the created pipe path on success.
```

Comment:
Should mention `f` has to implement  `io.ReadWriter`

Reply:
looks like it's either one or the other, maybe that's why it's not a `io.ReadWriter` in the first place?

Reply:
Oh yea yea my bad. Mention if `in` is true `f` should implement `io.Reader` and vice versa for `io.Writer` and `in` being false.

Reply:
Done